### PR TITLE
make it easier to choose a voice/language

### PR DESCRIPTION
**Pull request by zivoy**
Added a voices enum, so it is easier to select a language/voice for the TTS.
Also added the language to the path name to allow having the same text in multiple voices.

I added some of the languages to the enum, not all the ones that google translate can do

### CODE REVIEW

**Comment by hegedustibor** [comment]
Hey, thank you very much, cool PR. I will test and merge it.

File: htgotts_test.go [hunk 1]
```diff
@@ -1,14 +1,28 @@
 package htgotts
 
-import "testing"
-import "github.com/hegedustibor/htgo-tts/handlers"
+import (
+	"htgo-tts/handlers"
```
Comment (hegedustibor) [inline htgotts_test.go:4]: Please fix it:
github.com/hegedustibor/htgo-tts/handler

File: htgotts_test.go [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
+	"htgo-tts/voices"
```
Comment (hegedustibor) [inline htgotts_test.go:5]: Please fix it:
github.com/hegedustibor/htgo-tts/voices

**Review by hegedustibor — CHANGES_REQUESTED** [review]

**Comment by zivoy** [comment]
right sorry, those ware just because of the way my environment was set up

**Pull request merged**